### PR TITLE
Consistency

### PR DESCRIPTION
**Pull request by callum-oakley**
- [x] CHANGELOG updated if relevant?

### CODE REVIEW

File: src/chatkit.ts [hunk 1]
```diff
@@ -152,36 +209,50 @@ const TOKEN_EXPIRY_LEEWAY = 30;
...
 
   private tokenWithExpiry?: TokenWithExpiryAt;
 
   constructor(options: Options) {
     const { instanceLocator, key, port, host, client } = options;
 
-    const apiInstanceOptions = ({
+    const sdkInfo = new SDKInfo({
+      productName: 'chatkit',
+      version: '0.13.0', // TODO extract from package.json
```
Comment (vivangkumar) [inline src/chatkit.ts]: Is this TODO relevant for now?

Comment (callum-oakley) [reply src/chatkit.ts]: Hmmm, let me see if I can address it quickly. If it takes too long I'll leave it.

File: src/chatkit.ts [hunk 2]
```diff
@@ -219,7 +220,7 @@ export default class Chatkit {
 
     const sdkInfo = new SDKInfo({
       productName: 'chatkit',
-      version: '0.13.0', // TODO extract from package.json
+      version: packageJSON.version,
```
Comment (callum-oakley) [inline src/chatkit.ts]: @vivangkumar that was an enormous pain but after much trial and error this works 😅

File: src/chatkit.ts [hunk 3]
```diff
@@ -277,6 +349,14 @@ export default class Chatkit {
...
 
+  getUser(options: GetUserOptions): Promise<any> {
+    return this.apiInstance.request({
+      method: 'GET',
+      path: `/users/${encodeURIComponent(options.id)}`,
+      jwt: this.getServerToken(),
+    }).then(({ body }) => JSON.parse(body))
+  }
+
   getUsers(): Promise<any> {
```
Comment (hamchapman) [inline src/chatkit.ts]: This should support `fromTimestamp` (translated to `from_ts` in the query string) in its options I believe

Comment (callum-oakley) [reply src/chatkit.ts]: Hmmm yeah, well spotted. Will add it now.

**Pull request merged**